libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -24,15 +24,26 @@
 //! Mimic Qt's qApp for easy access to the application instance
 #define ccApp (static_cast<ccApplicationBase *>( QCoreApplication::instance() ))
 
+
+/*
+ * main ccApplicationBase that the CCViewer which implements
+ * QApplication: The QApplication class manages the GUI application's control flow and main settings
+ * https://doc.qt.io/qt-5/qapplication.html
+ * QApplication inherites QGuiApplication, which forces this app to use ::event(QEvent e)
+ *
+ * */
 class CCAPPCOMMON_LIB_API ccApplicationBase : public QApplication
 {
 public:
 	//! This must be called before instantiating the application class so it
 	//! can setup OpenGL first.
+    //! this is not from the QApplication parent class
```

Comment:
that's obvious otherwise we would use the 'override' keyword
Plus, additional comments should be in /** **/ sections (//! is for the brief description)